llabake/Hello-Books
# Update app packages

#### What does this PR do?
This PR updates app packages and Travis configuration

#### Description of Task to be completed?
- Configure prettier with Eslint
- Update node version in travis.yml
- Comment out client-side test command in travis.yml

#### How should this be manually tested?
* clone the repo
* cd into the project directory
* run `npm install` to install all the dependency
* run `npm run start-dev` to start the backend server
* run `npm run dev` to start the front-end server
* run `npm run test` all server-side test should be passing
#### What are the relevant pivotal tracker stories?
169250334

#### Any background context you want to add?
* test for the client-side is currently failing
#### Screenshots
N/A
